.vsts-pipelines/builds/PRBuild.yml

```diff
@@ -0,0 +1,232 @@
+phases:
+- template: ../templates/phases/default.yml
+  parameters:
+    name: 'UniverseCoherence'
+    buildArgs: >
```

Comment:
what does `>` do?  I'm more used to seeing `|`, or something like...

```JSON
buildArgs: $(PipeBuildArgs)
  /p:SkipTests=true
  /p:Configuration=Release
```

Perhaps `>` has the same affect of allowing multiline syntax but merging into a single line?  That would be neat, I'm just not familiar with it.